src/client/components/pages/Courses/CourseInstanceTable.tsx

```diff
@@ -182,6 +256,117 @@ const CourseInstanceTable: FunctionComponent<CourseInstanceTableProps> = ({
           <TableRow>
```

Comment:
As submitted, this is broken because this `<TableRow>` is only rendered when the enrollment fields are included, which is not true in the default view. Since the filters are always going to appear in the same row as the enrollment fields and we'll always have `area` and `course` columns with filters, we can probably remove all checks for `firstEnrollmentField > -1`

Reply:
This is fixed now so that the filters show whether or not the enrollment columns are showing.